src/mobile_manipulation_config/description/scene.xml

```diff
@@ -48,8 +21,22 @@
       type="plane"
       material="groundplane"
     />
+    <site
+      name="scene_camera_optical_frame"
+      pos="0.0 -2.0 3.5"
```

Comment:
@mkhansenbot I grabbed these numbers from [here](https://github.com/PickNikRobotics/moveit_studio_ur_ws/blob/main/src/mobile_manipulation_config/description/ur5e.xml#L207-L219), the `pos` does not match. The camera angle is way off too so I'm not eve sure if they are aligned. Do you have a different (working) camera frame?